__test__/app.test.js

```diff
@@ -61,7 +62,27 @@ describe('/api', () => {
             food_item_id: 1,
             name: 'New York Cheesecake',
             price: 4.5,
-            course: 'dessert'
+            course: 'dessert',
+            is_active: true,
+          });
+        });
+    });
+    it('SUCCESS status 201 - changes active state of a food item', () => {
+      return request(app)
+        .patch('/api/food-items/1')
+        .send({ food_item_id: 1,
```

Comment:
do we need to send the full food item if just patching the active state?

Reply:
we could just send the is_active state and it's a matter of preference for the frontend. What I had in mind was a user can see the all the details of food item and change where necessary and send request to backend.

Reply:
yeah I can see what you mean, food for thought for my patch request as well!